src/ensembl/src/shared/accordion/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,222 @@
+import { mount, ReactWrapper } from 'enzyme';
+import * as React from 'react';
+import {
+  Item,
+  Provider,
+  ProviderState
+} from '../AccordionContainer/AccordionContainer';
+import { default as AccordionItem } from '../AccordionItem/AccordionItem.wrapper';
+import { default as AccordionItemTitle } from '../AccordionItemTitle/AccordionItemTitle.wrapper';
+import { default as Accordion } from './Accordion.wrapper';
+
+describe('Accordion', () => {
+  it('different className', () => {
```

Comment:
let's write test descriptions in a grammatically sensible way? `describe` is a verb that takes a noun (`describe('Accordion')), but `it` is a noun that expects a predicate to make a clause (`it('accepts a class name')`, or something).

Instead of `describe('<Accordion accordion="true" />'`, why not describe what this `accordion=true` prop is representing (I don't know what it is, so can't offer any suggestions; perhaps you could explain?)

Reply:
Does the `accordion=true` prop allow opening only a single panel at a time?

Foundation's accordion calls this property "multi" (when it's on, multiple panels can be opened). Isn't it a better name?

Reply:
> Does the accordion=true prop allow opening only a single panel at a time?

Yes, that's right. I feel `multi' is still a bit vague. How about `allowMultiple`?